build/_k.shade

```diff
@@ -1,7 +1,7 @@
 @{/*
```

Comment:
shouldn't this file also be renamed to _dnx.shade?
same for other _k-*.shade files?